python/pyarrow/tests/test_pandas.py

```diff
@@ -2041,6 +2041,18 @@ def test_strided_objects(self, tmpdir):
         df = pd.DataFrame.from_dict(data)
         _check_pandas_roundtrip(df)
 
+    @pytest.mark.parametrize("typ", [
+        pa.decimal32,
+        pa.decimal64,
+        pa.decimal128,
+    ])
```

Comment:
Can you add `decimal256` into the mix, or does it actually not work?

Reply:
Sure, added `decimal256` as it appears to work